src/map/utils/guildutils.cpp

```diff
@@ -82,9 +82,10 @@ namespace guildutils
         }
         for (auto* PGuildShop : g_PGuildShopList)
         {
-            fmtQuery = "SELECT itemid, min_price, max_price, max_quantity, daily_increase, initial_quantity \
-                    FROM guild_shops \
-                    WHERE guildid = %u \
+            fmtQuery = "SELECT gs.itemid, gs.min_price, gs.max_price, gs.max_quantity, gs.daily_increase, gs.initial_quantity, ib.flags \
+                    FROM guild_shops gs \
+                    JOIN item_basic ib ON gs.itemid = ib.itemid \
+                    WHERE gs.guildid = %u \
```

Comment:
While you're here, can you change this query to use `db::preparedStmt` and change the multi-line query with trailing backslashes to just be multiple string literals:
```
"line 1 " // note the space
"line 2 "
"final line" // note the no space, and no semicolon